buildpipeline/Core-Setup-Publish.json

```diff
@@ -314,10 +429,23 @@
     },
     "TeamName": {
       "value": "DotNetCore"
+    },
+    "VsoToolsRepo": {
```

Comment:
So for `VsoToolsDir` (or other things the repo build uses), should we have `PB_VsoToolsDir` in the definition JSON and then an additional `VsoToolsDir` that just references `$(PB_VsoToolsDir)`?  Or can client repos just use the `PB_` variables?